src/Windowing/Silk.NET.Windowing/Silk.NET.Windowing.csproj

```diff
@@ -1,16 +1,27 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0</TargetFrameworks>
+        <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0;net6.0;net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
         <IncludeBuildOutput>false</IncludeBuildOutput> <!-- Metapackage -->
     </PropertyGroup>
 
     <ItemGroup>
         <ProjectReference Include="..\Silk.NET.Windowing.Common\Silk.NET.Windowing.Common.csproj" />
+    </ItemGroup>
+    
+    <ItemGroup Condition="'$(TargetFramework)' != 'net6.0-android' And '$(TargetFramework)' != 'net6.0-ios' And '$(TargetFramework)' != 'net6.0-maccatalyst' And '$(TargetFramework)' != 'net6.0'">
         <ProjectReference Include="..\Silk.NET.Windowing.Glfw\Silk.NET.Windowing.Glfw.csproj" />
         <ProjectReference Include="..\Silk.NET.Windowing.Sdl\Silk.NET.Windowing.Sdl.csproj" />
     </ItemGroup>
 
+    <ItemGroup Condition="'$(TargetFramework)' == 'net6.0-android' Or '$(TargetFramework)' == 'net6.0-ios' Or '$(TargetFramework)' == 'net6.0-maccatalyst'">
+        <ProjectReference Include="..\Silk.NET.Windowing.Sdl\Silk.NET.Windowing.Sdl.csproj" />
+    </ItemGroup>
+    
+    <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+        <ProjectReference Include="..\Silk.NET.Windowing.Glfw\Silk.NET.Windowing.Glfw.csproj" />
+    </ItemGroup>
```

Comment:
I'm sorry, *what*? Why? Why not just include both GLFW and SDL everywhere?

Reply:
Because we have no reason to. GLFW is priority one for desktop and there's zero reason to use SDL over GLFW on desktop - if for whatever reason you do want to use SDL you can go reference it yourself, but I don't think we should mandate that our users ship two extra (very large) DLLs (one for backend, and one for bindings) for a backend that they just don't need on their platform.

The SDL backend was only written to get us to platforms where GLFW doesn't work, so let's only use it on those platforms?